pkgs/development/python-modules/xarray/default.nix

```diff
@@ -1,34 +1,48 @@
 { lib
 , buildPythonPackage
 , fetchPypi
-, pytestCheckHook
 , numpy
 , pandas
+, pytestCheckHook
+, pythonOlder
 , setuptools
-, isPy3k
 , setuptools-scm
 }:
 
 buildPythonPackage rec {
   pname = "xarray";
-  version = "0.20.1";
-  disabled = !isPy3k;
+  version = "0.20.2";
+  format = "setuptools";
```

Comment:
a pyproject.toml is there, so it should use `pyproject` and `setuptoolsBuildHook` should then be added to the native build inputs

Reply:
Thanks, updated.